MADE.App.Design/MADE.App.Design.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;xamarin.ios10;monoandroid81;uap10.0.16299</TargetFrameworks>
+    <TargetFrameworks>netstandard1.4;xamarin.ios10</TargetFrameworks>
```

Comment:
Why was the Android & UWP references removed here?

Reply:
At present, there is no component in here that requires platform specifics for Android and Windows so these were removed. This package can still be installed as part of an Android or Windows project through the .NET Standard 1.4 reference. We can re-add in the future if we need platform specifics for Windows and Android.